Sources/PostgreSQL/Connection/PostgreSQLConnection+TCP.swift

```diff
@@ -1,11 +1,13 @@
 import Async
 import NIO
+import NIOOpenSSL
 
 extension PostgreSQLConnection {
     /// Connects to a Redis server using a TCP socket.
     public static func connect(
         hostname: String = "localhost",
         port: Int = 5432,
+        transportConfig: PostgreSQLTransportConfig = .cleartext,
```

Comment:
I would call this just `transport`, we have the strong type here anyway.

```swift
PostgreSQLConnection.connect(transport: .cleartext) 
```

That reads nicely.